src/linter/xmlTemplate/linter.ts

```diff
@@ -37,6 +37,11 @@ export default async function lintXml({filePathsWorkspace, workspace, context}:
 		await workspace.write(transpiledResource);
 		await filePathsWorkspace.write(transpiledResourceSourceMap);
 		await workspace.write(transpiledResourceSourceMap);
+
+		// Stash information that this .js file is actually a transpiled XML.
+		const contextMeta = context.getMetadata(jsPath);
+		contextMeta.xmlCompiledResources = contextMeta.xmlCompiledResources ?? new Set();
```

Comment:
Why storing a Set instead of just a flag that the resource has been transformed from XML (e.g. boolean)?
Isn't there always a 1:1 relation from XML source file to JS file?

Reply:
Done